Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -1004,6 +1004,33 @@ To remove all JavaScript source code and locally stored data, clear the WebView'
 
 Devices running platforms older than Android 4.4 (API level 19) use a version of WebKit that has several security issues. As a workaround, the app must confirm that WebView objects [display only trusted content](https://developer.android.com/training/articles/security-tips.html#WebView "WebView Best Practices") if the app runs on these devices.
 
+For more secure web browsing, Android 8.1 introduces an implementation for [`SafeBrowsing API`](https://developers.google.com/safe-browsing/v4). SafeBrowsing allows your application to detect URLs that Google has classified as a known threat.
+By default the webview shows a warning to users about the security risk with the option to load the URL or return to the safe page. With the SafeBrowsing API you can customize your application behavior by either reporting the threat to SafeBrowsing or performing a particular action such as returning back to safety each time it encounter a known threat.
+
+```Java
+public class MyWebViewClient extends WebViewClientCompat {
+    // Automatically go "back to safety" when attempting to load a website that
+    // Google has identified as a known threat. An instance of WebView calls
+    // this method only after Safe Browsing is initialized, so there's no
+    // conditional logic needed here.
+    @Override
+    public void onSafeBrowsingHit(WebView view, WebResourceRequest request,
+            int threatType, SafeBrowsingResponseCompat callback) {
+        // The "true" argument indicates that your app reports incidents like
```

Comment:
Move these comments down so that they are above the callback.backToSafety line

Reply:
Please do so :)

Reply:
I've checked and this example is the same as in Android Developers. I'd suggest to link to it directly and remove it from here for now. You can put something like: please check the [Android Developers documentation](link "title") for usage examples.

Reply:
@cpholguera @TheDauntless if you have no more suggestions i guess we could move the content to the MSTG-PLATFORM-2 section. i'll use the same branch for that.

Reply:
Hi @A-AFTAHI , thanks for the changes, please take a look at this message below:https://github.com/OWASP/owasp-mstg/pull/1655#pullrequestreview-361087479